lua/customroles/physician/sv_physician.lua

```diff
@@ -0,0 +1,84 @@
+--// Logan Christianson
+AddCSLuaFile()
+
+util.AddNetworkString("GetAllPhysicianTrackedPlayers")
+util.AddNetworkString("GetAllPhysicianTrackedPlayersCallback")
+
+local PHYSICIAN = PHYSICIAN or {tracking = {}}
+
+function PHYSICIAN:ResetAllTrackedPlayers()
+    PHYSICIAN.tracking = {}
+end
+
+function PHYSICIAN:AddNewTrackedPlayer(physician, trackedPly)
+    if physician:GetRole() ~= ROLE_PHYSICIAN then
+        return
+    end
+
+    local physicianSteamId = physician:SteamID64()
+
+    self.tracking[physicianSteamId] = self.tracking[physicianSteamId] or {physicianSteamId = 1}
```

Comment:
These `1`s and `2`s are enumeration values, I assume?
We normally define global enums for things like this to lend clarity.

See here for an example: https://github.com/Custom-Roles-for-TTT/TTT-Jingle-Jam-Roles-2022/blob/main/lua/customroles/faker.lua#L108